front/lib/front_web/templates/job/_status_badge.html.eex

```diff
@@ -0,0 +1,8 @@
+<div
+  class="flex mt1"
+  data-poll-background
+  data-poll-state="<%= @badge_pollman.state %>"
+  data-poll-href="<%= @badge_pollman.href %>"
+>
+  <%= raw job_status_badge(@job) %>
```

Comment:
instead of `raw job_status_badge` please put here the `<span>` from the view and call view helper just for for the class.